crates/move-stackless-bytecode/src/conditional_merge_insertion.rs

```diff
@@ -134,13 +134,17 @@ impl ConditionalMergeInsertionProcessor {
                 Bytecode::Assign(_, dst, src, _) => {
                     last_assign_per_var.insert(*dst, *src);
                 }
-                Bytecode::Call(_, dests, op, _, _) => {
-                    if Self::is_side_effecting_op(op) || dests.len() != 1 {
```

Comment:
Other passes insert trace calls. We rejected calls without a destination (including trace calls) for transformation. But these are harmless and should just be emitted as-is.